driver-core/src/test/unit/com/mongodb/connection/ElementExtendingBsonWriterSpecification.groovy

```diff
@@ -72,18 +75,18 @@ class ElementExtendingBsonWriterSpecification extends Specification {
         for (def cur : extraElements) {
             expectedDocument.put(cur.name, cur.value)
```

Comment:
And here's where the test was mutating what used to be a shared global singleton.